test/ClangImporter/cxx-exceptions-executable.swift

```diff
@@ -0,0 +1,39 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs/custom-modules/ -o %t/cxx_exceptions -Xfrontend -enable-cxx-interop
+// RUN: %target-codesign %t/cxx_exceptions
+// RUN: %target-run %t/cxx_exceptions
+
+import CxxExceptions
+import StdlibUnittest
+
+var CxxExceptionsTestSuite = TestSuite("CxxExceptions")
+
+// Uncaught C++ exceptions terminate the program.
```

Comment:
I'm not sure what this test shows -- it unwinds through a Swift frame, which we want to disallow.

Reply:
Maybe that's so, but this test can't "see" this.

The behavior that the test checks for (program is terminated if a C++ exception is not caught) will remain the same even when we've fixed Swift so it doesn't allow unwinding through Swift stack frames.